cmd/ore/aws/list-regions.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdListRegions = &cobra.Command{
+		Use:   "list-regions",
+		Short: "List enabled regions in the given AWS account",
+		Long: `List enabled regions in the AWS account and partition implied by the
+specified credentials file, profile, and region.`,
+		RunE: runListRegions,
+	}
+)
+
+func init() {
+	AWS.AddCommand(cmdListRegions)
+}
+
+func runListRegions(cmd *cobra.Command, args []string) error {
+	regions, err := API.ListRegions()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "could not list regions: %v\n", err)
+		os.Exit(1)
+	}
+
+	for _, region := range regions {
+		fmt.Println(region)
```

Comment:
WDYT about producing a JSON list rather than one region per line?

Reply:
No, not necessarily. I was more thinking from the perspective of this tool (in my eyes) mostly being useful in the context of being ran by scripts where I'd generally personally prefer to have a JSON output than parsing string outputs. I'm fine with leaving it as is if you feel it's too weighty for this tool.